airbrakes/hardware/imu.py

```diff
@@ -96,9 +96,8 @@ def get_imu_data_packets(self) -> collections.deque[IMUDataPacket]:
         # We use a deque because it's faster than a list for popping from the left
         data_packets = collections.deque()
         # While there is data in the queue, get the data packet and add it to the dequeue which we return
-        while not self._data_queue.empty():
-            data_packet = self.get_imu_data_packet()
-            data_packets.append(data_packet)
+        for _ in range(self._data_queue.qsize()):
```

Comment:
why change the for loop to a while loop?

Reply:
technically it saves repeated calls to .empty() every iteration